### PR TITLE
Add support for JWT Token authorization via query parameter

### PR DESCRIPTION
**Pull request by jiekang**
This adds support for extracting the JWT token from the query parameter. How does it look?

### CODE REVIEW

File: middleware/security/jwt/jwt.go [hunk 1]
```diff
@@ -134,6 +137,30 @@ func New(resolver KeyResolver, validationFunc goa.Middleware, scheme *goa.JWTSec
...
+
+	incomingToken := strings.Split(val, " ")[1]
+
+	return incomingToken, nil
+}
+
+func extractTokenFromQueryParam(schemeName string, req *http.Request) (string, error) {
+	incomingToken := req.URL.Query().Get(schemeName)
+	if incomingToken == "" {
+		return "", ErrJWTError(fmt.Sprintf("missing parameter %q %+v", schemeName, req))
```
Comment (raphael) [inline middleware/security/jwt/jwt.go]: Do we need to write the complete request object in the error? wondering if just writing the URL would be enough? what does this look like?

**Comment by raphael** [comment]
Thank you for the PR, looks great! I left one minor comment in the code.

File: middleware/security/jwt/jwt.go (hunk 1 above)
Comment (jiekang) [reply middleware/security/jwt/jwt.go]: Thank you for the catch! This printing of request object was not supposed to be there; I added it while I was trying to verify my tests.

**Comment by raphael** [comment]
Perfect, thank you! do you think you could backport this to `v1` as well? simply cherry-pick the commit in a branch created off of the `v1` branch and send a new PR. Thanks!

**Pull request merged**